cg/server/api.py

```diff
@@ -113,24 +97,25 @@ def families():
         records = db.cases_to_mip_analyze()
         count = len(records)
     else:
-        customer_obj = None if g.current_user.is_admin else g.current_user.customer
+        customer_objs = None if g.current_user.is_admin else g.current_user.customers
         case_query = db.families(
             enquiry=request.args.get("enquiry"),
-            customer=customer_obj,
+            customers=customer_objs,
             action=request.args.get("action"),
         )
         count = case_query.count()
         records = case_query.limit(30)
+
     data = [case_obj.to_dict(links=True) for case_obj in records]
```

Comment:
mmm data, don't we just love it! At least we could specify a little bit what the data is:
```suggestion
    data: List[dict] = [case_obj.to_dict(links=True) for case_obj in records]
```

Reply:
fixed